x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsTypeMappingDefinition } from '../../../../../src/core/server';
+
+export const alertMappings: SavedObjectsTypeMappingDefinition = {
+  properties: {
+    enabled: {
+      type: 'boolean',
+    },
+    name: {
+      type: 'text',
+      fields: {
+        keyword: {
+          type: 'keyword',
+          normalizer: 'lowercase',
```

Comment:
The addition of a new normalizer to the existing fields creates a mappings conflict:
```
Mapper for [alert.name.keyword] conflicts with existing mapper:
	Cannot update parameter [normalizer] from [null] to [lowercase]
```

However, I'm not sure if we should handle conflicts like that in code. I suppose it affects only dev environments but not users.